business/checkers/virtualservices/route_checker.go

```diff
@@ -125,13 +128,13 @@ func (route RouteChecker) checkTlsRoutes() ([]*models.IstioCheck, bool) {
 			}
 		}
 
-		trackTcpTlsSubset(routeIdx, "http", destinationWeights, &validations)
+		route.trackTcpTlsSubset(routeIdx, "http", destinationWeights, &validations)
```

Comment:
Here the "http" -> "tls", not for this PR, but it was a bug from the refactoring to use the Istio client, on this case, it's checking the tls routes and the string is used to check collissions.